zboss/include/osif/build_info.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef ZBOSS_BUILD_INFO_H__
+#define ZBOSS_BUILD_INFO_H__
+
+/**
+ * This is an autogenerated file.
+ * Please use release scripts to update it's contents.
+ *
+ * Repositories:
+ * platform_ncs       b1cffb1bc6668fa68ace58dc7711506c076f84b3 https://bitbucket-zoi.dsr-corporation.com/scm/ZOI/platform_ncs.git
+ * zboss-stack-src    8c5e5b4c7eb840b0f7ba8728fab1bf1bc1a0e6e3 https://bitbucket-zoi.dsr-corporation.com/scm/zoi/zboss_stable
+ * fw-nrfconnect-nrf  ed41097149389ae84438f6a16746359415c5f0e5 https://github.com/nrfconnect/fw-nrfconnect-nrf
```

Comment:
nrfxlib is listed with its new name, let's use the new name for this repo too:
`https://github.com/nrfconnect/sdk-nrf`

Reply:
This is actually a bug in `west.yml` file ;)